py/desitarget/mock/data/select-mock-targets-no-contam.yaml

```diff
@@ -0,0 +1,66 @@
+# Parameter file for desitarget/bin/select_mock_targets, including contaminants.
```

Comment:
The contaminants can just be commented out of the configuration file. That said, maybe it'd be worth adding a `--no-contaminants` optional input to `select_mock_targets` to overwrite whatever's in the config file.

Reply:
I always find myself doing this so such a flag, or yaml, would be useful in my opinion.